corehq/apps/callcenter/fixturegenerators.py

```diff
@@ -55,15 +55,10 @@ def __call__(self, restore_user, version, last_sync=None, app=None):
             return fixtures
 
         try:
-            fixtures.append(gen_fixture(restore_user, CallCenterIndicators(
-                domain.name,
-                domain.default_timezone,
-                domain.call_center_config.case_type,
-                restore_user
-            )))
```

Comment:
I didn't want to break the private `_` abstraction. also i wasn't sure yet how CallCenterIndicators acted with web users